cmd/mergesort/mergesort.go

```diff
@@ -226,14 +253,19 @@ func mergeSort(filename string, outFile string, numRecordsPerChunk int, sortCrit
 }
 
 func main() {
-	expectedNumArgs := 2
 	var numLinesPerChunk *int = flag.Int("tmpsize", 1000000, "The number of records to read into memory before writing to a tmp file.``")
 	var singleCellBx *bool = flag.Bool("singleCellBx", false, "Sort single-cell sam records by barcode.")
 	var sortCriteria string = "byGenomicCoordinates" // default the genomicCoordinates criteria.
+	var fastqPE *bool = flag.Bool("fastqPE", false, "Sort paired end fastq files. Each file will be sorted separately")
```

Comment:
Can we specify somewhere that these are being sorted by name? I notice that fastqFilter does not take in the sortCriteria string

Reply:
updated usage